index.html

```diff
@@ -7,15 +7,77 @@
     <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
     <link rel="stylesheet" href="css/normalize.css">
-    <!-- Add a link to your CSS file here (use the line above to guide you) -->
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <header>
+        <nav class="navbar"> 
+            <img class="logo" src="img\karma-logo.svg" alt="karma-logo">
+                
+                    
+                <a class="icon"><img src="img\menu-hamburger.svg" alt="Menu icon"> </a>
+                
+                    <div class="right">
+                            <a href="#" class="current-tab">Meet Karma</a>
+                            <a href="#">How it works</a>
+                            <a href="#">Store</a>
+                            <a href="#">Blog</a>
+                            <a href="#">Help</a>
+                            <a href="#">Login</a>
+                    </div>
+        </nav>
+        
+    </header>
+    <section class="intro">
+        <div class="content">
+            <h1>Introducing Karma</h1>
+            <h3>Bring WIFI with you, everywhere you go</h3>
+            <div>
+                <button type="button" class="button">Learn More</button>
+            </div>
+        </div>
+    </section>
+    <div class="statement">
+        <h1>Everyone needs a little Karma.</h1>
+    </div>
+    <section class="main">
+        <div class="devices">
+            <img src="img\icon-devices.svg" alt="Multiple Devices">
+            <h3 class="statement-for-icons">
+                Internet for all devices
+            </h3>
+        </div>
+        <div class="coffee">
+            <img src="img\icon-coffee.svg" alt="coffee mug icon">
+            <h3 class="statement-for-icons">
+                Boost your productivity
+            </h3>
+        </div>
+        <div class = "refill">
+            <img src="img\icon-refill.svg" alt="pay as you go">
+            <h3 class="statement-for-icons">
+                Pay as You Go
+            </h3>
+        </div> 
+    </section>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+<hr class="solid">
 
 </body>
+<footer>
+    <h3 class="statement"> Join us on</h3>
+        <section class="footer">
+        <div class="twitter-icon">
+            <img src="img\twitter-icon.svg" alt="Twitter">
+        </div>
+        <div class="facebook-icon">
+            <img src="img\facebook-icon.svg" alt="Facebook">
+        </div>
+        <div class="instagram-icon">
+            <img src="img\instagram-icon.svg" alt="Instagram">
+        </div>
+        </section>
+    <h4 class="statement"> © Karma Mobility Inc</h4>
```

Comment:
This is probably not a heading since it's a tiny bit of text. You can use `p` or `span` for this one.

Reply:
Updated HTML